tests/test_other.py

```diff
@@ -5133,92 +5133,90 @@ def test_libcxx_minimal(self):
     # we can strip out almost all of libcxx when just using vector
     self.assertLess(2.25 * vector, iostream)
 
-  def test_minimal_dynamic(self):
-    def run(wasm):
-      print('wasm?', wasm)
-      library_file = 'library.wasm' if wasm else 'library.js'
+  @parameterized({
+    '': (True,),
+    # TODO(sbc): make dynamic linking work with wasm2js
```

Comment:
side note, I think it's probably not worth looking into that TODO before doing some general work on wasm2js, that would make it easier. wasm2js output is already pretty close to a "polyfill" for real wasm, and making it even closer would make stuff like this just work, basically.

Reply:
My read on this TODO is that wasm2js needs to be improved, yes.